sdk/attestation/attestation/src/utils/base64.browser.ts

```diff
@@ -84,10 +84,34 @@ export function hexToByteArray(value: string): Uint8Array {
   return Uint8Array.from(byteArray);
 }
 
+export function byteArrayToHex(value: Uint8Array): string {
+  return value.reduce((str, byte) => str + byte.toString(16).padStart(2, "0"), "");
+}
+
 /**
  * Converts a hex encoded string to its base64 equivalent.
  * @param value - Hex encoded value
  */
 export function hexToBase64(value: string): string {
   return base64EncodeByteArray(hexToByteArray(value));
 }
+
+export enum PemType {
+  Certificate = "CERTIFICATE",
+  PrivateKey = "PRIVATE KEY"
+}
+/**
+ *
+ * @param base64 - Base64 encoded DER object to encode as PEM.
+ * @param pemType - PEM object type - typically "CERTIFICATE" |
+ */
+export function pemFromBase64(base64: string, pemType: PemType): string {
```

Comment:
consider factoring these duplicated definitions out in a common file?

Reply:
Already done.